products/identityplatform/api.yaml

```diff
@@ -0,0 +1,408 @@
+          hyphens, underscores or periods. The part after 'saml.' must also start with a lowercase letter, end with an
+          alphanumeric character, and have at least 2 characters.
+        required: true
+      - !ruby/object:Api::Type::String
+        name: 'displayName'
+        description: |
+          Human friendly display name.
+        required: true
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        description: |
+          If allows users to sign in with the provider.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'idpConfig'
+        required: true
+        description: |
+          SAML IdP configuration when the project acts as the relying party
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'idpEntityId'
+            required: true
+            description: |
+              Unique identifier for all SAML entities
+          - !ruby/object:Api::Type::String
+            name: 'ssoUrl'
+            required: true
+            description: |
+              URL to send Authentication request to.
+          - !ruby/object:Api::Type::Boolean
+            name: 'signRequest'
+            description: |
+              Indicates if outbounding SAMLRequest should be signed.
+          - !ruby/object:Api::Type::Array
+            name: 'idpCertificates'
+            required: true
+            description: |
+              The IdP's certificate data to verify the signature in the SAMLResponse issued by the IDP.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+              - !ruby/object:Api::Type::String
+                name: 'x509Certificate'
+                description: |
+                  The IdP's x509 certificate.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'spConfig'
+        required: true
+        description: |
+          SAML SP (Service Provider) configuration when the project acts as the relying party to receive
+          and accept an authentication assertion issued by a SAML identity provider.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'spEntityId'
+            description: |
+              Unique identifier for all SAML entities.
+          - !ruby/object:Api::Type::String
+            name: 'callbackUri'
+            description: |
+              Callback URI where responses from IDP are handled. Must start with `https://`.
+          - !ruby/object:Api::Type::Array
+            name: 'spCertificates'
+            output: true
+            description: |
+              The IDP's certificate data to verify the signature in the SAMLResponse issued by the IDP.
+            item_type: !ruby/object:Api::Type::NestedObject
+              properties:
+              - !ruby/object:Api::Type::String
+                name: 'x509Certificate'
```

Comment:
I think this also should be marked output

Reply:
Ah, that setting doesn't get inherited?